### ISSUE TITLE
Slim throw Notices 

### ISSUE THREAD

**Issue by jcubic** [slimphp/Slim#386]
These are the messages that are thrown by develop branch

```
Notice: Use of undefined constant MCRYPT_RIJNDAEL_256 - assumed 'MCRYPT_RIJNDAEL_256' in /srv/www/staging.openclipart.org/htdocs/libs/Slim/Slim/Slim.php on line 233
Notice: Use of undefined constant MCRYPT_MODE_CBC - assumed 'MCRYPT_MODE_CBC' in /srv/www/staging.openclipart.org/htdocs/libs/Slim/Slim/Slim.php on line 234
Notice: Use of undefined constant MCRYPT_RIJNDAEL_256 - assumed 'MCRYPT_RIJNDAEL_256' in /srv/www/staging.openclipart.org/htdocs/libs/Slim/Slim/Slim.php on line 233
Notice: Use of undefined constant MCRYPT_MODE_CBC - assumed 'MCRYPT_MODE_CBC' in /srv/www/staging.openclipart.org/htdocs/libs/Slim/Slim/Slim.php on line 234
```

**Comment by jcubic** [comment]
These linens where removed from Slim.php but those constants are still in use and there are not defined in any file. 

``` php
if ( !defined('MCRYPT_RIJNDAEL_256') ) {
    define('MCRYPT_RIJNDAEL_256', 0);
}
if ( !defined('MCRYPT_MODE_CBC') ) {
    define('MCRYPT_MODE_CBC', 0);
}
```

Will add this to my first pull request.

**Comment by codeguy** [comment]
libmcrypt is a system requirement. I want to reduce Slim's global namespace footprint which is why this code was originally removed.

**Comment by jcubic** [comment]
I added those lines to my PR I can add another commit to remove them again.

**Comment by chelmertz** [comment]
@codeguy This is kinda annoying :) I can't run tests on my bare-minimum local dev machine because of this (notices are caught by phpunit).

And "reducing Slim's global namespace footprint" does not equal "silently rely on dependencies being met and crash in the middle of everything" to me :) Composer might not be ideal quite yet but I would be pretty annoyed if I was allowed to _install_ an rpm/deb but it crashed when I tried to _run_ it.

**Comment by jcubic** [comment]
I fixed this by installing libmcrypt

**Comment by codeguy** [comment]
I've fixed this in 1.6.7. Pull the latest code and it'll work again as it did before.

**Comment by chelmertz** [comment]
Wow, that was fast. Great job!
